package.json

```diff
@@ -113,6 +113,21 @@
 					"type": "string",
 					"default": "",
 					"description": "The scene file to run"
+				},
+				"godot_tools.reconnect_automatically": {
+					"type": "boolean",
+					"default": true,
+					"description": "Whether the plugin should attempt to reconnect"
+				},
+				"godot_tools.reconnect_cooldown": {
+					"type": "number",
+					"default": 1000,
```

Comment:
This seems pretty frequent, if I'm understanding it right. With 10 retries, it means the editor will only try to reconnect for 10 seconds (or perhaps 11).

Maybe this should be set to 3000 ms by default.